pkgs/misc/deepspeech/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  name = "deepspeech-${version}";
+  version = "0.9.3";
+
+  src = fetchurl {
+    url = "https://github.com/mozilla/DeepSpeech/releases/download/v${version}/native_client.amd64.cpu.linux.tar.xz";
```

Comment:
Can we build from source?

Reply:
Ideally yes, but it would be complex and I have no time to implement it at this point.